mmhuman3d/core/conventions/keypoints_mapping/spin_smplx.py

```diff
@@ -0,0 +1,30 @@
+'''SPIN in smplx convention.
+SPIN_SMPLX_KEYPOINTS can be found in 
+https://github.com/vchoutas/expose
+'''
+SPIN_SMPLX_KEYPOINTS = [
```

Comment:
What is the difference between `SPIN_SMPLX_KEYPOINTS` and `SMPL_49` in smpl.py?

Reply:
Also add the corner case to this [issue](https://github.com/open-mmlab/mmhuman3d/issues/83).